test/APITests.js

```diff
@@ -71,20 +70,15 @@ describe('Simple HiGlassComponent', () => {
 
       waitForTransitionsFinished(api.getComponent(), () => {
         waitForTilesLoaded(api.getComponent(), () => {
-          // console.log(api.getComponent().yScales.a.domain());
 
           expect(api.getComponent().yScales.a.domain()[0]).to.be.above(2);
 
           const trackObj = api.getTrackObject('a', 'heatmap1');
-          // console.log('trackObj', trackObj);
-          // trackObj.getMouseOverHtml(50,50);
 
           const rd = trackObj.getVisibleRectangleData(285, 156, 11, 11);
           expect(rd.data.length).to.eql(1);
-          // trackObj.getVisibleRectangleData(24,8,11,11);
 
-          // console.log('visi', trackObj.visibleAndFetchedTiles());
-          document.body.removeChild(div);
+          removeHGComponent(div);
```

Comment:
Here and below, maybe put it in an afterEach, rather than needing to remember to include it on every test. (And if every test wouldn't use the same afterEach, rearrange so they would?)

Reply:
Good idea. Changed.